src/components/checkbox/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { memo } from 'react';
+
+import { RequiredKeys } from '@typings/index';
+
+type InputProps = RequiredKeys<
+  React.DetailedHTMLProps<
+    React.InputHTMLAttributes<HTMLInputElement>,
+    HTMLInputElement
+  >,
+  'id'
+>;
+
+const Checkbox: React.FC<InputProps> = ({ ...props }) => (
+  <label
+    className="w-7 w-7 flex justify-center items-center rounded-lg border-2 
+  border-solid border-black border-gray-600 cursor-pointer overflow-hidden"
+  >
+    <input type="checkbox" {...props} className="peer hidden" />
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
```

Comment:
Save the svg in `/public/icons/` and render it as a component. Example [here](https://github.com/Rostand-Corp/flitch-frontend/blob/main/src/layouts/sign-up/index.tsx#L9)